pkg/controller/operators/olm/operator.go

```diff
@@ -257,6 +257,27 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 			return nil, err
 		}
 
+		// Register QueueInformer for olmConfig
+		olmConfigInformer := externalversions.NewSharedInformerFactoryWithOptions(
+			op.client,
+			config.resyncPeriod(),
+		).Operators().V1().OLMConfigs().Informer()
+		olmConfigQueue := workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), fmt.Sprintf("%s/olm-config", namespace))
+		olmConfigQueueInformer, err := queueinformer.NewQueueInformer(
+			ctx,
+			queueinformer.WithInformer(olmConfigInformer),
+			queueinformer.WithLogger(op.logger),
+			queueinformer.WithQueue(olmConfigQueue),
+			queueinformer.WithIndexer(olmConfigInformer.GetIndexer()),
+			queueinformer.WithSyncer(queueinformer.LegacySyncHandler(op.syncOLMConfig).ToSyncer()),
+		)
+		if err != nil {
+			return nil, err
+		}
+		if err := op.RegisterQueueInformer(olmConfigQueueInformer); err != nil {
+			return nil, err
+		}
```

Comment:
The OLMConfig controller needs to requeue csvs, which is done by adding an object to [this queue](https://github.com/operator-framework/operator-lifecycle-manager/blob/e6cc305d49e865020b0d3481b621bf7412fb871a/pkg/controller/operators/olm/operator.go#L71). We could build and pass this queue into OLM if we wanted the olmConfig controller to be built with controller-runtime. Thoughts?